### DISCUSSION TITLE
¿Corregir <img src="{{ thumbnail }} width="{{ thumbnail.width }}" height="{{ thumbnail.height }}">?

### DISCUSSION THREAD

**Discussion by AlanJair** [pimcore/pimcore#14533] [Q&A / Support]
En la sección "Image Editable" de la documentación aparece `<img src="{{ thumbnail }} width="{{ thumbnail.width }}" height="{{ thumbnail.height }}">` como ejemplo para mostrar una imagen derivada de su thumbnail, pero por lo que veo `<img src="{{ thumbnail }}">` ya lo redimensiona sin necesidad de `width="{{ thumbnail.width }}"` o  `height="{{ thumbnail.height }}"`

¿Esto representa una modificación/actualización en la documentación porque su ejemplo es obsoleto o me equivoco y quizás aun no me doy cuenta de que no es obsoleto?

**Comment by NiklasBr** [comment]
The attributes `width` and `height` are often included in `<img>` tags [to avoid browsers needing to redo the layout](https://stackoverflow.blog/2022/12/27/picture-perfect-images-with-the-modern-element/) once the image has been downloaded.
